tools/dotnet-linker/LinkerConfiguration.cs

```diff
@@ -13,6 +13,7 @@
 namespace Xamarin.Linker {
 	public class LinkerConfiguration {
 		public List<Abi> Abis;
+		public string AssemblyName { get; private set; }
```

Comment:
The naming is unclear, that's the _root_ or _entry_ assembly right ?

Reply:
It's the `AssemblyName` MSBuild property of the executable project, so yes, it's the root/entry assembly. I'll add a comment to explain it a bit better.